tests/processes/Processes.Worker.Library.Tests/ProcessExecutorTests.cs

```diff
@@ -80,10 +80,10 @@ public void GetRegisteredProcessTypeIds_ReturnsExpected()
     public async Task ExecuteProcess_WithInvalidProcessTypeId_Throws()
     {
         // Arrange
-        var Act = async () => await _sut.ExecuteProcess(Guid.NewGuid(), (ProcessTypeId)default, CancellationToken.None).ToListAsync().ConfigureAwait(false);
+        async Task Act() => await _sut.ExecuteProcess(Guid.NewGuid(), (ProcessTypeId)default, CancellationToken.None).ToListAsync();
```

Comment:
async + await are redunant - please remove
to fix the codeql-finding: (ProcessTypeId)default -> default(ProcessTypeId)

Reply:
adjusted as discussed